packages/studio-plugin/src/createStudioStylingPlugin.ts

```diff
@@ -1,8 +1,9 @@
 import { PluginOption } from "vite";
+import isStyleFile from "./utils/isStyleFile";
 
 /**
- * Adds the `?inline` query parameter to all CSS imports
- * unless they are CSS imported by Studio. Studio CSS files are
+ * Adds the `?inline` query parameter to all CSS and SCSS imports
+ * unless they are imported by Studio. Studio CSS files are
```

Comment:
nit: since you specifically call out CSS and SCSS imports separately above, the mention of just CSS here makes me think the `studioStyling` query param would only be on CSS files

Reply:
changed to "Studio styling"